app/views/govuk_component/docs/related_items.yml

```diff
@@ -0,0 +1,45 @@
+name: "Related items"
+description: "Headed sections of related items."
+body: |
+  Accepts an array of sections. Each section can have a title, url, id and a list of related items.
+
+  Each item is a hash with a title and url. If the url is external, a rel value can also be provided.
+fixtures:
+  default:
+    sections:
+    - title: "Travel Abroad"
+      url: "/"
+      id: "related-travel-abroad"
+      items:
+      - title: "Link A"
+        url: /
+      - title: "Link B"
+        url: /
+    - title: "Travel"
+      id: "related-travel"
+      items:
+      - title: "Link 1"
+        url: /
+      - title: "Link 2"
+        url: /
+  external_links:
+    sections:
+      - title: "Elsewhere on the web"
+        id: "related-elsewhere-on-the-web"
+        items:
+        - title: "Wikivorce"
+          url: "http://www.wikivorce.com"
+          rel: external
```

Comment:
:+1: those make sense, possibly `rel="up"` too.

I was thinking that in a mustache environment, if we were to do something conditional on a specific value of `rel` then we'd have issues, but thats theoretical, and it wouldn't be a hard API chance to accommodate that.